source/_integrations/sensor.rest.markdown

```diff
@@ -290,22 +290,17 @@ sensor:
 {% raw %}
 
 ```yaml
-sensor:
-  - platform: rest
-    name: JSON time
-    json_attributes:
-      - date
-      - milliseconds_since_epoch
-    resource: http://date.jsontest.com/
-    value_template: "{{ value_json.time }}"
-  - platform: template
-    sensors:
-      date:
-        friendly_name: "Date"
-        value_template: "{{ state_attr('sensor.json_time', 'date') }}"
-      milliseconds:
-        friendly_name: "milliseconds"
-        value_template: "{{ state_attr('sensor.json_time', 'milliseconds_since_epoch') }}"
+rest:
+  - resource: http://date.jsontest.com/
```

Comment:
This needs to be quoted, as it is a string.

See: <https://developers.home-assistant.io/docs/documenting/yaml-style-guide#strings>

Reply:
https://github.com/home-assistant/home-assistant.io/pull/27593